app/Http/Controllers/UsersController.php

```diff
@@ -75,7 +75,23 @@ public function edit(User $user)
     public function update(Request $request, User $user)
     {
 
+        $user->fill([
+            'name'     => $request->name,
+            'email'    => $request->email,
+            'password' => bcrypt($request->password),
+        ]);
+
+        if($request->hasFile('avatar')) {
+            $avatar = $request->file('avatar');
+            $filename = time() . '.' . $avatar->getClientOriginalExtension();
+            Image::make($avatar)->resize(300,300)->save( public_path('/uploads/avatars/' . $filename) );
+
+            $user->avatar = $filename;
+        }
```

Comment:
you can put this on User Model class, since this involve in database interaction, its better that model will handle this code.

Reply:
okaay